### DISCUSSION TITLE
Add config to move comment in start position

### DISCUSSION THREAD

**Discussion by tapacko88** [spatie/laravel-sql-commenter#10] [Ideas]
In the current project, we use very long queries (long query size and long execution time) and we have a page or somebody use applications where we can see all queries in real time, like 
SHOW FULL PROCESSLIST, 
but usually, we can see only the first few hundred characters of a query and would be good to see comments and not load the full queries and find where is comment.

Is it possible to add this option?

Thanks.

**Comment by riasvdv** [comment]
This is not according to the standard (https://google.github.io/sqlcommenter/) and I don't think all clients would support this so it's not something we'll change. You could try doing this in a fork of the package of your own.
